blog/2023-10-24-apache-enters-apache-incubator/index.md

```diff
@@ -0,0 +1,97 @@
+
+Since the first release, Answer has had a total of 19 releases, 7.9K+ GitHub Stars, and over 10,000 downloads. We have worked in collaboration with 37 code contributors and 59 volunteers from around the world, receiving a total of 221 pull requests. Answer has also been translated into 15 different languages.
+
+The Initial Committers of the Answer project are listed as follows, and big thanks to  their significant contributions to the early phase of the Answer project:
+- Ning Qi https://github.com/joyqi
+- Feng Dong https://github.com/fenbox
+- Fengjun Lv https://github.com/mingcheng
+- Enxin Xie https://github.com/LinkinStars
+- Guangfu Yang https://github.com/kumfo
+- Yubin Ren https://github.com/robinv8
+- Shuailing Li https://github.com/shuashuai
+- Haitao Jia https://github.com/haitaoo
+- Chenyang Ai https://github.com/aichy126
+- Bo Jiang  https://github.com/jiangbonadia
+- Anna Primma https://github.com/PrimmaAnna
+
+About Apache Incubator [1]
+![Alt text](apache-incubator-logo.png)
+
+Created in 2002, the Apache Incubator provides services to projects which want to enter the Apache Software Foundation (ASF). It helps those incoming projects (called "podlings") adopt the Apache style of governance and operation and guides them to the ASF services available to our projects, so they can become top-level ASF projects ("TLPs").
+
+The Incubator delegates a few mentors for each podling to act as liaisons with the various ASF teams: Incubator PMC, Infrastructure team, etc., and facilitate the podling’s growth and operations.
+
+Answer will embrace the Apache way with the guidance of the 4 Incubator Mentors:
+- Willem Jiang, Director of the Apache Software Foundation and Incubator Mentor.
+- tison, Member of the Apache Software Foundation and Incubator Mentor.
+- Justin Mclean, Apache Software Foundation VP, and Board Member.
+- Christofer Dutz, Board Member of the Apache Software Foundation and Incubator Mentor.
+
+## Acknowledgement
+Joining Apache Incubator is a significant milestone of Answer and marks new challenges and beginnings. We would love to express our profound gratitude to our Champion Willem Jiang, as well as our mentors tison, Justin Mclean, and Christofer Dutz, for their support and encouragement during the proposal. 
+
+We would also love to give a big applause and thank you to contributors and volunteers around the world for the contributions and support.
+
+
+## Join Answer Community
+Answer's power comes from the talented contributers in its community. While we celebrate this moment, we would also like to invite more contributors to be a part of our Apache journey. If you are interested in this project, please check out and join us.
+
+Visit Answer's official website: http://answer.apache.org/     
+Explore and contribute to our GitHub: https://github.com/apache/incubator-answer       
+Check our documentation: https://answer.apache.org/docs          
+Join our official community: https://meta.answer.dev/         
```

Comment:
@gmcdonald @Humbedooh how do you think of hosting a forum site with ASF INFRA? Is it possible currently?

@PrimmaAnna I suppose we should move this forum to apache.org instead of hosting an "official" forum in a third-party domain.

Reply:
also cc @chrisdutz @justinmclean @WillemJiang

Reply:
> @gmcdonald @Humbedooh how do you think of hosting a forum site with ASF INFRA? Is it possible currently?
> 
> @PrimmaAnna I suppose we should move this forum to apache.org instead of hosting an "official" forum in a third-party domain.

I understand this, let me loop @fenbox here as well.